imports/plugins/included/taxes-taxcloud/client/settings/taxcloud.js

```diff
@@ -20,8 +22,23 @@ Template.taxCloudSettings.helpers({
 AutoForm.hooks({
   "taxcloud-update-form": {
     onSuccess: function () {
-      return Alerts.toast(i18next.t("admin.taxSettings.shopTaxMethodsSaved"),
-        "success");
+      if (!TaxCodes.findOne({ taxCodeProvider: "taxes-taxcloud" })) {
+        Meteor.call("taxcloud/getTaxCodes", (err, res) => {
+          if (err) {
+            throw new Meteor.Error(500, "description");
```

Comment:
We no longer recommend using number error codes. Also "description should be an actual description of the error.

The suggested template for errors is
```
throw new Meteor.Error("short-description", "Longer description")
```

E.g. 
```
throw new Meteor.Error("operation-failed", "client safe message about why")
```